### PR TITLE
Constant estimation 

### PR DESCRIPTION
**Pull request by PatrikValabek**
Data for different constants related to the type of plant are generated. Example use of how to generate this data. 

Also Salad Model Parameters Study from Jozef Vargan. Based on this study constants were chosen.

### CODE REVIEW

**Comment by MarekWadinger** [comment]
Hey @PatrikValabek 👋,

nice work, 👏 I will review it tomorrow. 

After quick look, only one thing captured my attention, the copy of lettuce model `core/constant_est_lettuce_model.py`. For easier maintenance of the code, we should avoid such copies.

As all the constants are available in module space, it is possible to overwrite them from within script calling `core/constant_est_lettuce_model.py` and the change will be reflected within its functions, similarly to changing them as globals.

If you couldn't figure out how, I could contribute it tomorrow as well.

**Comment by MarekWadinger** [comment]
Hey @PatrikValabek 👋

please, kindly review my suggested modifications to this pull request. 

Overall, very important feature which will further increase impact of our project 👏☺️

I proposed some changes to the structure, maintainability and extensibility of constants estimation, mainly:

**moved results from data/ -> results/constant_estimation/**
(by default all in results/ is ignored, I would consider whether do we need pkls to be published as they are more than 2MB each and each of their commited changes may potentially increase size of git history by 2MB, which is a lot in the long run!)

**modified original lettuce_model**
I see what is the challenge of constant estimation from perspective of their modification. The `lettuce_model`had some compatibility with constant changes but was not ready yet. However, selecting some constants and giving user a tuple argument without knowledge of their order is not practical nor scalable.
I proposed using kwargs to modify any or all the constants, exposing them in new function `get_default_constants`. This way (considering that constants have meaningful upper case value), user both knows about possible constants and have ability to change them. In the examples, I had to change using map of length similar to number of expected constants in tuple to a tool modifying default dict of constants or any dict where constants are supposed to be.

File: core/simulate.py
Comment (MarekWadinger) [inline core/simulate.py:1]: Passing down kwargs is useful not only for constants :)

File: main.ipynb
Comment (MarekWadinger) [inline main.ipynb:1]: Should implement notebook cleanup before commit as of https://stackoverflow.com/questions/73218304/how-to-clear-jupyter-notebooks-output-and-metadata-when-using-git-commit

File: results/constant_estimation/all_100.pkl
Comment (MarekWadinger) [inline results/constant_estimation/all_100.pkl:1]: In general, we should avoid publishing result data, which could be consistently and replicably acquired by anyone. Please consider deleting pkls.

Comment (MarekWadinger) [reply results/constant_estimation/all_100.pkl:1]: Anyways, seems like all parameter modification was planned, so current status allows to do so easily. :)

File: results/constant_estimation/growth_parameters.pdf
Comment (MarekWadinger) [inline results/constant_estimation/growth_parameters.pdf:1]: Useful, I moved it along but might be beneficial to have this in form of README for constant estimation. :)

File: core/lettuce_model.py
Comment (MarekWadinger) [inline core/lettuce_model.py:1]: This version allows modifying any or all constants

**Review by MarekWadinger — APPROVED** [review]

**Comment by PatrikValabek** [comment]
In the def randomize_const() there was min and max changes in % set to min = 0.9 and max = 0.1. I suppose there was a typo, because, if we want change to be of maximum of 10%, max should be 1.1? Am I right or does the constants mean something else?

**Review by MarekWadinger — APPROVED** [review]
Good catch, it was a typo.

**Review by MarekWadinger — APPROVED** [review]
Formated and sanitized

**Pull request merged**